funding/manager.go

```diff
@@ -2162,6 +2162,15 @@ func (f *Manager) continueFundingAccept(resCtx *reservationWithCtx,
 		f.failFundingFlow(resCtx.peer, pendingChanID, err)
 		return
 	}
+
+	// With FundingCreated sent, we can now notify Brontide to keep track
+	// of this pending open channel.
+	err = resCtx.peer.AddPendingChannel(channelID, f.quit)
```

Comment:
Ah ok, makes sense, this is just an active funding attempt that _may_ become a channel in the future. I think we could update some of the `godoc` comments to make that a bit more clear (basically this is where we stage things).